### DISCUSSION TITLE
[QUESTION] Compatibility with PrusaSlicer 2.4.2

### DISCUSSION THREAD

**Discussion by Grimaction** [sn4k3/Ender3#42] [Q&A] [answered]
Hi,

Could you fix a need for "G92 E0" after each layer?

**Comment by sn4k3** [comment, marked as answer]
Printer settings - Custom G-code - Remove the G92 E0

![prusa-slicer_2022-05-31_17-31-38](https://user-images.githubusercontent.com/113281/171226485-a4983d8e-9b07-4ce6-bcb4-62f1885b631d.png)
